NeoMathEngine/src/CPU/arm/CpuArmMathEngineVectorMathPrivate.h

```diff
@@ -910,6 +910,40 @@ inline void vectorHSwish( const float* first, float* result, int vectorSize )
 	}
 }
 
+//------------------------------------------------------------------------------------------------------------
+
+inline float32x4_t vectorHardSigmoidWorker( const float32x4_t& val,
+	const float32x4_t& zero, const float32x4_t& one, const float32x4_t& slope, const float32x4_t& bias )
+{
+	float32x4_t mainVal = vaddq_f32( vmulq_f32( val, slope ), bias );
```

Comment:
Here it will be enough scalar values for `slope` and `bias` (use vmulq_n_f32 and vaddq_n_f32 intrinsics)

Reply:
Right now we don't use MVE/Helium.

Reply:
It's not MVE-specific, IMHO.

Reply:
It's unexpected that `vmulq_n_f32` is NEON but `vaddq_n_f32` is VME. I've checked `vaddq_n_f32` first and concluded that `mul` will be in the same set.